dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,212 @@
+        image: highlight-abbyomg
+      - title: Zomerfeest
+        description: Heet weer en nog hetere mensen! DJ Maarten verzorgde tropische tunes terwijl we zomercocktails aan het nippen waren en afkoelde in een daadwerkelijk zwembad!
+        image: highlight-summer-party
+</i18n>
+
+<script setup>
+useHead({
+  bodyAttrs: {
+    class: 'mixup-colors',
+  },
+})
+
+const { t, tt } = useT()
+
+const { data: openingHours } = await useAsyncData(() => queryContent('opening_hours').findOne())
+const barOpeningHours = openingHours.value.events.find((event) => event.day.en === 'Saturday')
+
+const { image: imageOverviews } = useDynamicImages(import.meta.glob('~/assets/images/photos/mixup/*', { eager: true }))
+
+const instagramChannelsMixup = [
+  {
+    name: 'MIXUP',
+    widgetId: '219aedf13a4355fe95c6e27ed1f7386c',
+    instagram: 'mixupdelft',
+  },
+]
+</script>
+
+<template>
+  <LayoutSmallHeader triangleClass="border-brand-450">
+    {{ t('title') }}
+  </LayoutSmallHeader>
+
+  <section>
+    <PagesMixupLinkedEvents />
+  </section>
+
+  <section>
+    <div class="bg-brand-900 text-white text-lg mx-auto pt-12 pb-24 md:flex">
+      <div class="flex-1 px-4 lg:pr-32">
+        <div class="space-y-4">
+          <ElementsParagraphedText :paragraphs="t('intro')" class="md:text-xl md:leading-relaxed space-y-4" />
+          <p
+            v-if="barOpeningHours.announcement"
+            class="mt-3 mb-4 text-brand-900 bg-brand-450"
+            v-text="tt(barOpeningHours.announcement)"
+          />
+          <div
+            class="flex-1 rounded-lg shadow-xl bg-brand-450 md:text-xl p-4 relative w-full z-20 md:w-auto md:inline-flex items-center"
+          >
+            <div class="pt-6 md:pt-0 md:pl-4 space-y-2">
+              <p>{{ t('invite.announcement') }}</p>
+              <p class="font-bold">{{ t('invite.time', [barOpeningHours.start_time]) }}</p>
+            </div>
+          </div>
+          <div class="flex flex-1 space-x-4">
+            <div>
+              <a href="https://my.dwhdelft.nl/signup">
+                <ElementsSecondaryButton class="!text-brand-450" arrow>
+                  {{ t('membership_button') }}
+                </ElementsSecondaryButton>
+              </a>
+            </div>
+            <div>
+              <nuxt-link :to="localePath('barbuddy')">
+                <ElementsSecondaryButton class="!text-brand-450" arrow>
+                  {{ t('barbuddy_button') }}
+                </ElementsSecondaryButton>
+              </nuxt-link>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="hidden lg:block">
+        <div>
+          <img src="~/assets/images/photos/mixup/barnight.png" class="w-96" />
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mixup-colors">
+    <div class="bg-[url('~/assets/images/photos/mixup/barvisual.jpg')] bg-cover bg-center py-10">
+      <ElementsContainer>
```

Comment:
Bonus points for making the bulletpoints component of the out-site reusable, and reusing it here, so we avoid duplication

Reply:
(ik bedoelde dat je bonuspunten kunt verdienen als we dat doen 🙈 )

Reply:
Ah, nee, die las ik even verkeerd. Done :)